migrations/20241107115400_count_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE envelopes
```

Comment:
Did you test the migration locally?

Reply:
Yes. And it's fast:

> The ALTER TABLE command works by modifying the SQL text of the schema stored in the [sqlite_schema table](https://www.sqlite.org/schematab.html). No changes are made to table content for renames or column addition without constraints. Because of this, the execution time of such ALTER TABLE commands is independent of the amount of data in the table and such commands will run as quickly on a table with 10 million rows as on a table with 1 row.

https://www.sqlite.org/lang_altertable.html